<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChanegInfoDialog</class>
 <widget class="QDialog" name="ChanegInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>601</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>601</width>
     <height>150</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>头像</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cb_icon">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>110</height>
       </size>
      </property>
      <property name="maxVisibleItems">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>600</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>200</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>昵称</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_name">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>60</height>
       </size>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>个性签名</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="le_feeling">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>60</height>
       </size>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
